src/audio/dai-zephyr.c

```diff
@@ -384,17 +356,17 @@ static int dai_comp_get_hw_params(struct comp_dev *dev,
 				  int dir)
 {
 	struct dai_data *dd = comp_get_drvdata(dev);
-	int ret;
+	const struct dai_config *cfg = dai_config_get(dd->dai->dev, dir);
 
 	comp_dbg(dev, "dai_hw_params()");
 
 	/* fetching hw dai stream params */
-	ret = dai_get_hw_params(dd->dai, params, dir);
```

Comment:
the comment above needs to be removed/changed as well?

Reply:
ok